runtime/bcverify/rtverify.c

```diff
@@ -1437,6 +1440,13 @@ verifyBytecodes (J9BytecodeVerificationData * verifyData)
 					/* Jazz 82615: Save the location of receiver */
```

Comment:
in this part of the code it can only be putstatic, putfield or withfield. So you can do something like `bc != JBputstatic` since saving the location of the receiver is common for both putfield and withfield